### DISCUSSION TITLE
how to import all size from one font (noto-sans-sc) at once

### DISCUSSION THREAD

**Discussion by aboveyunhai** [fontsource/fontsource#227] [Q&A] [answered]
I have trouble to import `noto-sans-sc` inside the current `next.js` project.
The first line only contains the default `normal` or `400` after `build`. As a result I have to import all of them one by one as following.

```
import "@fontsource/noto-sans-sc";  
import "@fontsource/noto-sans-sc/100.css";
import "@fontsource/noto-sans-sc/300.css";
import "@fontsource/noto-sans-sc/500.css";
```
I just check out the doc website that you are able to do the font import like the follow, which seems like not available for `noto-sans-sc`.
https://github.com/fontsource/fontsource/blob/8d9983750a0e2dadd91e01d783313fa41dc9d29e/website/src/pages/_app.tsx#L2

Is there anything I miss from the doc?

**Comment by Glenn-Mendonca** [comment]
I guess to import all weights u might consider something like this if I have got u right.
@import url('https://fonts.googleapis.com/css2?family=Noto+Sans+SC:wght@100;300;400;500;700;900&display=swap');
Pls revert if u have a different issue. :)

**Comment by ayuhito** [comment, marked as answer]
Hey @aboveyunhai,

`import "@fontsource/raleway/variable.css";` is part of the [variable font](https://fontsource.org/docs/variable-fonts) specification which is only supported by a limited number of fonts. This does not include `noto-sans-sc`.

We split it by weight to ensure you're only loading CSS that you need and nothing unnecessary. It only needs to be imported once in `_document.tsx` after all and Webpack will handle the rest. :)
